src/riak_kv_util.erl

```diff
@@ -160,6 +161,20 @@ expand_value(Type, default, BucketProps) ->
 expand_value(_Type, Value, _BucketProps) ->
     Value.
 
+expand_sync_on_write(default, BucketProps) ->
+    normalize_value(get_bucket_option(sync_on_write, BucketProps));
+expand_sync_on_write(Value, _BucketProps) ->
+    Value.
```

Comment:
should you not normalise here as well?

Reply:
I think normalize_value may be unnecessary in both cases due to validation at the API/riak_kv_bucket